starboard/android/shared/player_get_preferred_output_mode.cc

```diff
@@ -82,7 +82,8 @@ SbPlayerOutputMode SbPlayerGetPreferredOutputMode(
         is_sdr) {
       return kSbPlayerOutputModeDecodeToTexture;
     }
-    SB_LOG(ERROR) << "Invalid output mode due to play HDR video on sub player";
+    SB_LOG_IF(ERROR, !is_sdr)
```

Comment:
INFO instead of ERROR, as SbPlayerGetPreferredOutputMode() is a function for query.

Reply:
Done.